docs/content/reference/infrastructure/aws.md

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
Do we need the image above? I don't see where it's used?

Reply:
nope in this PR. Thanks